### DISCUSSION TITLE
Cannot find module '../../hooks/use-sync-refs.js'

### DISCUSSION THREAD

**Discussion by faltawy** [tailwindlabs/headlessui#2444] [Help]
Unhandled Runtime Error
Error: Cannot find module '../../hooks/use-sync-refs.js'

Call Stack
webpackMissingModule
webpack-internal:///./node_modules/@headlessui/react/dist/components/combobox/combobox.js (14:50)
eval
webpack-internal:///./node_modules/@headlessui/react/dist/components/combobox/combobox.js (14:153)
./node_modules/@headlessui/react/dist/components/combobox/combobox.js
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/pages/settings.js (747:1)
options.factory
/_next/static/chunks/webpack.js (713:31)
__webpack_require__
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (368:21)
eval
webpack-internal:///./node_modules/@headlessui/react/dist/headlessui.esm.js (3:90)
./node_modules/@headlessui/react/dist/headlessui.esm.js
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/pages/settings.js (923:1)
options.factory
/_next/static/chunks/webpack.js (713:31)
__webpack_require__
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (368:21)
eval
webpack-internal:///./pages/settings/index.tsx (8:75)
./pages/settings/index.tsx
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/pages/settings.js (522:1)
options.factory
/_next/static/chunks/webpack.js (713:31)
__webpack_require__
file:///E:/dentofolio%20foundation/dentofoliofrontendo/.next/static/chunks/webpack.js (37:33)
fn
/_next/static/chunks/webpack.js (368:21)
eval
node_modules\next\dist\build\webpack\loaders\next-client-pages-loader.js?absolutePagePath=E%3A%5Cdentofolio%20foundation%5Cdentofoliofrontendo%5Cpages%5Csettings%5Cindex.tsx&page=%2Fsettings! (5:15)
execute
node_modules\next\dist\client\route-loader.js (211:50)

**Comment by harrisonratcliffe** [comment]
Ever find a fix for this?
